CRM/Contribute/Form/ContributionBase.php

```diff
@@ -220,10 +220,35 @@ class CRM_Contribute_Form_ContributionBase extends CRM_Core_Form {
 
   /**
    * Is the price set quick config.
+   *
    * @return bool
    */
-  public function isQuickConfig() {
-    return self::$_quickConfig ?? FALSE;
```

Comment:
@seamuslee001 this knocks off another php8 bug - although my goal here is actually to try to disentangle the code enough I can figure out how to fix up the assigns for the membership receipt it does allow me to fix some of these while I'm at it